src/main/fc/rc_modes.h

```diff
@@ -78,7 +78,12 @@ typedef enum {
     BOXUSER4         = 49,
     BOXCHANGEMISSION = 50,
     BOXBEEPERMUTE    = 51,
-    BOXMULTIFUNCTION = 52,
+#if defined(USE_VARIABLE_PITCH)    
```

Comment:
Do we need to handle flight modes here?

Or  should we have modes to switch governor profiles instead?

Reply:
flight modes also tend to imply handling pitch curves, btw. That is probably better left for the radio

Reply:
> Do we need to handle flight modes here?
> 
> Or should we have modes to switch governor profiles instead?

For the moment I am using this to switch hover settings (`nav_hc_hover_collective_normal`, `nav_hc_hover_collective_idleup_1`, `nav_hc_hover_collective_idleup_2`) as well as the target RPM for the governor, if governor type SET is configured (available governor types are `OFF` | `SIMPLE` | `SET`).

Curves are handled by the radio.